account_invoice_reminder/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+##############################################################################
+#
+#    OAC, Business Open Source Solution
+#    Copyright (C) 2013-2016 Open Architects Consulting SPRL.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Simple Payment reminder",
+    "version": "12.0.1.0.0",
+    "depends": ["account"],
+    "author": "Houssine BAKKALI <houssine.bakkali@gmail.com>",
```

Comment:
It's also bad practice to remove authorship. Is this accurate?

Reply:
the current policy here is to set “Coop IT Easy SCRLfs” as author for our own modules, and set all contributors in `CONTRIBUTORS.rst`.